packages/atomic/src/utils/utils.ts

```diff
@@ -84,3 +84,14 @@ export function parseAssetURL(url: string, assetPath = './assets') {
   }
   return null;
 }
+
+// TODO: add tests
+export function elementHasAncestorTag(
```

Comment:
Could the following method be used instead https://developer.mozilla.org/en-US/docs/Web/API/Element/closest ?

Reply:
I tried that but it didn't work, can't remember why now! will have another look